### DISCUSSION TITLE
Could I remap my laptop function row to an external QMK enabled keyboard?

### DISCUSSION THREAD

**Discussion by Zylon676** [seerge/g-helper#3165] [Q&A]
I was think about buy an external monitor setup for my g-15 (2022), and I do switch between fan modes quite a bit is it posible to remap my external keyboard special function to my laptop such as Fn+F5 for fan control to the external board.

**Comment by seerge** [comment]
@Zylon676 you don't need to remap anything. `CTRL + SHIFT + F5` is [already reserved for that on any keyboard](https://github.com/seerge/g-helper?tab=readme-ov-file#%EF%B8%8F-keybindings)
